src/darwin/mod.rs

```diff
@@ -319,13 +319,13 @@ impl Font {
         let average_advance = self.glyph_advance('0');
 
         let ascent = self.ct_font.ascent() as f64;
-        let descent = self.ct_font.descent() as f64;
+        let descent = self.ct_font.descent().floor() as f64;
```

Comment:
Why did you floor here? Where is this coming from?

Reply:
I've looked that freetype always gives integers, but macOS provide float, and this float doesn't make any sense, how descend coordinate could be 4.5(it should be negative, but it seems we use some weird api)? For us it just useless, so I did like it looks in freetype backend. Without that we have underline being bleed into font, and with that floor it looks like in `Terminal.app` for the same font.